src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/WebClientConfig.kt

```diff
@@ -12,9 +14,11 @@ import reactor.netty.http.client.HttpClient
 import java.time.Duration.ofSeconds
 
 @Configuration
-class WebClientConfig {
+class WebClientConfig(
+  @Autowired private val metricsCustomizer: MetricsWebClientCustomizer
+) {
 
-  @Bean
+  @Bean // health check only, not logging metrics
```

Comment:
Would be preferable to avoid a WebClient @Bean which is only good for /health?  
If not avoidable then rename to e.g. `fun webClientNoAuthNoMetrics()`?